recipes/tests.rb

```diff
@@ -303,6 +303,16 @@ module load intelmpi && mpirun --help | grep 'Version 2019 Update 8'
   end
 end
 
+###################
+# EFA - GDR
+###################
+if node['conditions']['efa_supported'] && efa_gdr_enabled?
```

Comment:
this test is not triggered unless `efa_gdr_enabled` is enabled from the kitchen suite